include/openssl/ocsp.h

```diff
@@ -19,10 +19,14 @@
 extern "C" {
 #endif
 
-/* Various OCSP flags and values */
-#  define OCSP_NOINTERN                   0x2
-#  define OCSP_NOCHAIN                    0x8
-#  define OCSP_NOEXPLICIT                 0x20
+// Various OCSP flags and values
+
+// Set OCSP_NOCERTS for OCSP_request_sign() if no certificates are included
+// in the |OCSP_REQUEST|. Certificates are optional.
+#define OCSP_NOCERTS                    0x1
+#define OCSP_NOINTERN                   0x2
+#define OCSP_NOCHAIN                    0x8
+#define OCSP_NOEXPLICIT                 0x20
```

Comment:
What is required for constant define documentation?

Reply:
The other three were probably missed out in previous imports. I can add documentation for them.